docs/landing-zone-v2/README.md

```diff
@@ -99,7 +99,7 @@ This can be deployed once or multiple times per organization as highlighted in t
 
 ### Client Landing Zone
 
-This section is made up of two packages [client-setup](../../solutions/client-setup/) and [client-landing-zone](../../solutions/client-landing-zone/). The client setup package prepares the Config Controller instances for the new client and creates the required service accounts and namespaces for the client to use.
+This section is made up of two packages [client-setup](../../solutions/client-setup/) and [client-landing-zone](../../solutions/client-landing-zone/). The client setup package prepares the Config Controller instances for the new client and creates the required service accounts and namespaces for the client to use.  Both of these client packages must be installed using the [GitOps](#3-deploy-the-infrastructure-using-gitops) approach
```

Comment:
Missing a period at the end of the last sentence, after approach.

Reply:
thanks luc - missed that - refactored the md changes to point further in the flow to client-onboarding instead.